### PR TITLE
[15.0] [MIG] project_category->project_type

### PR DESCRIPTION
**Pull request by Murtaza-OSI**
Ref:- [project_category](https://github.com/OCA/project/pull/872) 
For Rename Module

### CODE REVIEW

**Comment by leemannd** [comment]
Hello, thank you for the work. Seems good unless the pre-commit.

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Comment by CRogos** [comment]
I thinks the Tests should be renamed as well? Not sure for the translations files. [PR](https://github.com/ursais/project/pull/4).

Is this module-renaming also working in case of updating a database from v14 to v15, or will there be an error that project_category is missing? I'm asking because I have similar issue [here](https://github.com/OCA/timesheet/pull/507).

File: project_type/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module adds the category type fields in project and task.
```
Comment (gurneyalex) [inline project_type/readme/DESCRIPTION.rst:1]:
```suggestion
This module adds the type field to record the type of project in the project and task models.
```

File: project_type/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import test_project_category
```
Comment (gurneyalex) [inline project_type/tests/__init__.py]: rename the test file

File: project_type/tests/test_project_category.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Onestein
```
Comment (gurneyalex) [inline project_type/tests/test_project_category.py]: file to be renamed

**Review by gurneyalex — CHANGES_REQUESTED** [review]
Additionnally to the requested changes above, please add a migration script.

**Comment by CRogos** [comment]
@Murtaza-OSI are you still working on this topic?

**Comment by Murtaza-OSI** [comment]
> @Murtaza-OSI are you still working on this topic?
IT already done

**Comment by CRogos** [comment]
Have you seen my PR https://github.com/ursais/project/pull/4 a while ago? Also the test file was not renamed?

**Comment by Murtaza-OSI** [comment]
> Have you seen my PR [ursais#4](https://github.com/ursais/project/pull/4) a while ago? Also the test file was not renamed?

Thank for your help.
I merge your PR

**Review by Chanakya-OSI — APPROVED** [review]

**Comment by CRogos** [comment]
Can we add an empty project_category module, with this module as dependency, so that the upgrade is done correctly? Or is there any other suggestion for the transition v14->v15?
I did something similia here [hr_timesheet_activity_begin_end\\__manifest__.py](https://github.com/OCA/timesheet/blob/230c728283b7fbabcb9af8ea658909413811273e/hr_timesheet_activity_begin_end/__manifest__.py).
Or is there any better way?

**Review by leemannd — APPROVED** [review]

**Comment by tobehn** [comment]
Is this merged and brought to mainline v15 already?
I'd love to see it available.
Thank you so much 🙏

**Comment by leemannd** [comment]
Hello @gurneyalex, your comments have been attended. Could you update your review?

**Comment by CRogos** [comment]
Could you also have a look at this improvement for upgrade https://github.com/ursais/project/pull/5

**Review by gurneyalex — APPROVED** [review]
thanks

**Pull request merged**